test/test_faker_bsn.rb

```diff
@@ -0,0 +1,16 @@
+require File.expand_path(File.dirname(__FILE__) + '/test_helper.rb')
+
+class TestFakerBsn < Test::Unit::TestCase
+  def test_bsn_length
+    bsn = Faker::Bsn.bsn
+    assert [8,9].include? bsn.to_s.length
+  end
+
+  def test_bsn_validity
+    100.times do
```

Comment:
i added 100 times because its a random number because this method is not idempotent.